accounts/accounts/routes/ui.py

```diff
@@ -84,6 +84,22 @@ def unset_submission_cookie(response: Response) -> None:
     response.set_cookie('submit_session', '', max_age=0, httponly=True)
 
 
+def unset_permanent_cookie(response: Response) -> None:
+    """
+    Users who elect a permanent cookie expect it to be unset when they log out.
+
+    If it is not unset, legacy components will attempt to log them back in.
+    """
+    permanent_cookie_name = current_app.config['CLASSIC_PERMANENT_COOKIE_NAME']
+    domain = current_app.config['AUTH_SESSION_COOKIE_DOMAIN']
+    response.set_cookie(permanent_cookie_name, '', max_age=0, expires=0,
```

Comment:
I'm not sure how cookies work with flask but usually expires is suppose to be a date. Sending zero might be working but it that might be unspecified behavior.
For an example see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie

Reply:
I think Flask supports UNIX timestamp, but probably good to switch to datetime.now(UTC) if just for clarity